scipy/ndimage/tests/test_fourier.py

```diff
@@ -9,7 +9,7 @@
 
 class TestNdimageFourier:
 
-    @pytest.mark.parametrize('shape', [(32, 16), (31, 15)])
+    @pytest.mark.parametrize('shape', [(32, 16), (31, 15), (1, 10)])
     @pytest.mark.parametrize('dtype, dec',
                              [(numpy.float32, 6), (numpy.float64, 14)])
     def test_fourier_gaussian_real01(self, shape, dtype, dec):
```

Comment:
Is there a reason you only add this to the real tests?

Reply:
I figured one test for each of the 3 python functions was enough.